docs/News.md

```diff
@@ -16,6 +16,33 @@
     
 *   **Gene Tables Update:** Updated tables of genes (main and alias), based on [Oct 1, 2023 HGNC release](http://ftp.ebi.ac.uk/pub/databases/genenames/hgnc/archive/monthly/tsv/). See seedDB release notes [here](https://github.com/cBioPortal/datahub/tree/master/seedDB#latest-seed-database-schema-2131) for details.
 
+## Dec 5, 2023
+* **New feature**: Filter by data types on the homepage:
+    ![image](https://github.com/cBioPortal/cbioportal/assets/1334004/cf6d5486-bfd7-4f69-add0-442dbb4d354a)
+* **New feature**: Change colors of tracks in Oncoprint:
+    <img src="https://github.com/cBioPortal/cbioportal/assets/1334004/301b0efb-3063-44db-bc4c-86973e7c566a" width="700" />
+* **New feature**: Show alteration frequencies per group in OncoPrint:
+    [Example: RTK-RAS alterations in Smokers vs Never Smokers Lung Cancer Cases in MSK-IMPACT Clinical Sequencing Cohort](https://bit.ly/3PwKsDJ)
+    <img src="https://github.com/cBioPortal/cbioportal/assets/1334004/f2a9576e-c06d-447a-8eac-f0292bd1f935" width="700" />
+
+## Oct 17, 2023
+* **New feature**: Survival charts with landmark events and hazard ratios:
+    [Example: TP53 in Lung Cancer Cases in MSK-IMPACT Clinical Sequencing Cohort](https://bit.ly/48QktxS)
+    <img src="https://github.com/cBioPortal/cbioportal/assets/1334004/dc97c3ae-ebd7-4cd3-bb76-165517c9d2ba" width="700" />
```

Comment:
Same thing, the preview is showing "Sequencing Cohort" as a second line of text along with the image, which means there's lots of vertical space between the first & second lines of text. Can you make the image go below all the text?